src/build.rs

```diff
@@ -357,6 +357,12 @@ impl BuildQueue {
                                 cmd_dep_info.arg(a);
                             }
                         }
+                        // Compilation may depend on env vars which Cargo sets during compile-time
+                        for (k, v) in cmd.get_envs() {
```

Comment:
Would
```
cmd_dep_info.envs(cmd.get_envs()
.iter()
.map(|(k, v)| (k, v.unwrap_or(OsString::from("")))));
```
inline be better?
I wanted to retain a value Option and couldn't think of a clean way to do it, but on a second thought providing a default value seems reasonable in this case.